study.md

```diff
@@ -151,7 +151,12 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+<!-- CRUD
```

Comment:
This is correct, but I think they were looking for the actual verbs that are used in HTTP: GET, POST, PATCH, DELETE